part2/stages/Functions/install-main

```diff
@@ -591,7 +594,7 @@ create_efi_boot_entries()
 
 copy_to_esp()
 {
-    local ESP=$(sfdisk -l -q | grep "EFI System" | awk '{ print $1 }')
+    local ESP=$(sfdisk -l -q "/dev/${TARGET_DISK}" | grep "EFI System" | awk '{ print $1 }')
```

Comment:
Is `TARGET_DISK` defined during an OTA/upgrade?

Reply:
No it's not, good point. Working on a fix.

Reply:
Updated.